File: fizz.r
for (i in 0:99) {
  if (!i %% 15) {
    print("FizzBuzz")
  } else if (!i %% 5) {
    print("Buzz")
  } else if (!i %% 3) {
    print("Fizz")
  } else {
    print(i)
  }
}

File: fizz.php
<?php
for ($i=0; $i < 100; $i++) {
  if ($i % 15 == 0) {
    echo "FizzBuzz\n";
  } elseif ($i % 5 == 0) {
    echo "Buzz\n";
  } elseif ($i % 3 == 0) {
    echo "Fizz\n";
  } else {
    echo $i . "\n";
  }
}
?>
